src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -2641,12 +2641,6 @@ private static async Task<string> CloneRestoreAndBuild(string path, Job job, str
                 channel = "edge";
             }
 
-            // Until there is a "current" version of net7.0, use "edge"
-            if (targetFramework.Equals("net7.0"))
-            {
-                channel = "edge";
-            }
-
```

Comment:
Can we change this block including from 2633 so we can use what is passed on the command line? This won't break the current behavior (use edge for net7.0 if nothing is specified) while letting you force net7.0 or RC versions

```c#
            // If a specific channel is set, use it instead of the detected one
            if (!String.IsNullOrEmpty(job.Channel))
            {
                channel = job.Channel;
            }
            // Until there is a GA version of net7.0 or net8.0, use "edge"
            else if (targetFramework.Equals("net7.0") || targetFramework.Equals("net8.0"))
            {
                channel = "edge";
            }
```